extensions/fetch/26_fetch.js

```diff
@@ -331,6 +331,9 @@
         }
       }
     }
+    if (request.method === "HEAD" || request.method === "OPTIONS") {
+      request.body = null;
+    }
```

Comment:
This should happen in `mainFetch`, not in `httpRedirectFetch`.

Reply:
Also you are setting the request body, but the relevant spec says to set the _response_ body to null.

Reply:
thanks